docker/dev/Dockerfile

```diff
@@ -5,66 +5,16 @@ FROM ubuntu:18.04
 # http://us.archive.ubuntu.com/ubuntu/, or if you are in China, you
 # can try https://mirrors.tuna.tsinghua.edu.cn/ubuntu/
 ARG APT_MIRROR="http://us.archive.ubuntu.com/ubuntu/"
-RUN echo "\n\
-deb $APT_MIRROR bionic main restricted universe multiverse \n\
-deb $APT_MIRROR bionic-security main restricted universe multiverse \n\
-deb $APT_MIRROR bionic-updates main restricted universe multiverse \n\
-deb $APT_MIRROR bionic-proposed main restricted universe multiverse \n\
-deb $APT_MIRROR bionic-backports main restricted universe multiverse \n\
-" > /etc/apt/sources.list
+ENV APT_MIRROR=$APT_MIRROR
+COPY install.sh /
+RUN /install.sh
```

Comment:
As many dependencies are in the CI image now, so I merged software installations into one file, `install.sh`.